### ISSUE TITLE
Core dump on empty input

### ISSUE THREAD

**Issue by Jellyfrog** [jtheoof/swappy#105]
Getting a core dump when feeding swappy with a empty input like so: `swappy -f - -o -`, this also happens when using it with grim but pressing escape before selecting area 

```
#0  0x00007ffff7cf5ba8 in g_log_structured_array () at /usr/lib/libglib-2.0.so.0
#1  0x00007ffff7cf5e76 in g_log_default_handler () at /usr/lib/libglib-2.0.so.0
#2  0x00007ffff7cf72b9 in g_logv () at /usr/lib/libglib-2.0.so.0
#3  0x00007ffff7cf7560 in g_log () at /usr/lib/libglib-2.0.so.0
#4  0x000055555555cf7d in pixbuf_init_from_file ()
#5  0x000055555555d5d5 in  ()
#6  0x00007ffff7b4b658 in  () at /usr/lib/libgio-2.0.so.0
#7  0x00007ffff7de2ddf in g_closure_invoke () at /usr/lib/libgobject-2.0.so.0
#8  0x00007ffff7e0bfbd in  () at /usr/lib/libgobject-2.0.so.0
#9  0x00007ffff7dff2db in g_signal_emit_valist () at /usr/lib/libgobject-2.0.so.0
#10 0x00007ffff7e00210 in g_signal_emit () at /usr/lib/libgobject-2.0.so.0
#11 0x00007ffff7ba9d27 in  () at /usr/lib/libgio-2.0.so.0
#12 0x00007ffff7bab00a in  () at /usr/lib/libgio-2.0.so.0
#13 0x00007ffff7bab19a in g_application_run () at /usr/lib/libgio-2.0.so.0
#14 0x0000555555559092 in main ()
```

**Comment by Arnavion** [comment]
This was already fixed by https://github.com/jtheoof/swappy/commit/cdbd06d7af94b4aedfc2bda2231da8853f775f3a

**Comment by jtheoof** [comment]
I just released [1.4.0](https://github.com/jtheoof/swappy/releases/tag/v1.4.0) which contains the fix.
